articles/d-array-article.dd

```diff
@@ -148,21 +148,21 @@ $(H2 $(LNAME2 determinism, Determinism))
 
 $(P There is one caveat with slice appending that can bite inexperienced, and even experienced D coders: the apparent non-deterministic behavior of appending.)
 
-$(P Let's say we have a function which is passed a buffer, and writes some number of A's to the buffer (appending if necessary), returning the filled buffer:)
+$(P Let's say we have a function which is passed a buffer, and writes some number of 'A's to the buffer (appending if necessary), returning the filled buffer:)
```

Comment:
Here we can add a leading apostrophe to make it correct because we're talking about the character `'A'`.